server/controllers/user.js

```diff
@@ -0,0 +1,23 @@
+const showUser = (req, res, next) => {
+  // check for logged in user
+  const userId = req.user ? req.user.id : null;
+  if (!userId)
+    return res
+      .status(401)
+      .send("You are not authorized to access this resource");
+  try {
+    const user = User.findById(userId).select("-password").exec();
```

Comment:
Take a look at https://mongoosejs.com/docs/api.html#document_Document-toJSON and https://mongoosejs.com/docs/api.html#document_Document-toObject - so that we always remove password when we convert user to Object or to JSON

Reply:
Thanks! I'll put together a quick patch with a toObject transform on the schema.